client/cliClient.py

```diff
@@ -48,14 +48,36 @@ def _add_response(response: Packet) -> str:
     code = ResponseCode(data['responseCode'])
     if code == ResponseCode.OK:
         return "Successfully added you to the schedule!"
+    elif code == ResponseCode.CONFLICT:
+        return "Can't work on the same day as " + ', '.join(data['results'])
+    elif code == ResponseCode.UNEXPECTED:
+        return "Unsuccessfully added you to the schedule"
+    elif code == ResponseCode.FORBIDDEN:
+        return "Something has gone terribly wrong"
 
 
 def _remove_response(response: Packet) -> str:
-    pass
+    data = response.data
+    code = ResponseCode(data['responseCode'])
+    if code == ResponseCode.OK:
+        return "Successfully removed you from the schedule!"
+    elif code == ResponseCode.NOT_FOUND:
```

Comment:
Use full sentences, all sentences should end in punctuation. That means periods when we aren't !!!!ng them.
In this case "You were not scheduled for this day, and therefore did not need to be removed." (or something along those lines). We want to give as much info as possible.